kuma/wiki/admin.py

```diff
@@ -260,12 +260,26 @@ def rendering_info(obj):
 
 
 def akismet_data_as_dl(akismet_data):
+    """Format Akismet data as a definition list."""
+    favorites = (
```

Comment:
I think its better to have this ordering in a dictionary rather than list. So it will be easily configurable.
Moreover, it should live in ``constants`` so we can configure it easily